salt/utils/cloud.py

```diff
@@ -1356,18 +1356,28 @@ def deploy_windows(host,
                 salt.utils.smb.delete_directory('salttemp', 'C$', conn=smb_conn)
         # Shell out to psexec to ensure salt-minion service started
         if use_winrm:
+            
             winrm_cmd(winrm_session, 'sc', ['stop', 'salt-minion'])
-            time.sleep(5)
+            time.sleep(10)
             winrm_cmd(winrm_session, 'sc', ['start', 'salt-minion'])
         else:
+            
             stdout, stderr, ret_code = run_psexec_command(
                 'cmd.exe', '/c sc stop salt-minion', host, username, password
             )
             if ret_code != 0:
                 return False
 
-            time.sleep(5)
-
+            stdout, stderr, ret_code = run_psexec_command(
+                'cmd.exe', '/c sc query salt-minion', host, username, password
+            )
+            log.info('%s',stdout) 
```

Comment:
I don't think we need this log statement.

Reply:
I will change to debug cause I think it's informative for debugging.